controllers/pipeline/pipeline_adapter.go

```diff
@@ -385,11 +385,13 @@ func (a *Adapter) prepareSnapshotForPipelineRun(pipelineRun *tektonv1beta1.Pipel
 	snapshot.Labels[gitops.SnapshotTypeLabel] = gitops.SnapshotComponentType
 	snapshot.Labels[gitops.SnapshotComponentLabel] = a.component.Name
 
-	// copy PipelineRun PAC annotations/labels from Build to snapshot
+	// Copy PipelineRun PAC annotations/labels from Build to snapshot.
+	// Modify the prefix so the PaC controller won't react to PipelineRuns generated from the snapshot.
 	pipelinerunLabels := pipelineRun.GetLabels()
 	for key, value := range pipelinerunLabels {
-		if strings.Contains(key, "pipelinesascode.tekton.dev") {
-			snapshot.Labels[key] = value
+		if strings.Contains(key, "pipelinesascode.tekton.dev/") {
+			newKey := strings.ReplaceAll(key, "pipelinesascode.tekton.dev/", "test.appstudio.openshift.io/")
```

Comment:
Should we use `test.appstudio.openshift.io/` like this or further subdivide the namespace to avoid accidentally overriding labels/annotations? For example, if PaC adds a `pipelinesascode.tekton.dev/component` label at some point it's likely to collide with `test.appstudio.openshift.io/component`? Here are a few ideas:

- `pac.test.appstudio.openshift.io/`
- `pipelinesascode.test.appstudio.openshift.io/`
- `pipelinesascode.tekton.dev.test.appstudio.openshift.io/`

Thoughts?

Reply:
We're aiming to reduce the number of `test.*` labels with [HACBS-1331](https://issues.redhat.com/browse/HACBS-1331), but your point still stands about the possibility of collision.
The `pac.test.appstudio.openshift.io/` looks like the best option to me on account of brevity.

Reply:
Done. All the PaC labels/annoations use the `pac.test.appstudio.openshift.io/` prefix now.

Reply:
Thanks!